src/time_bucket.c

```diff
@@ -291,3 +291,63 @@ ts_time_bucket_by_type(int64 interval, int64 timestamp, Oid timestamp_type)
 
 	return ts_time_value_to_internal(time_bucketed, timestamp_type);
 }
+
+/*
+ * Add one bucket to a time value.
+ *
+ * The time value is typically the start of a time bucket, but need not be.
+ */
+Datum
+ts_time_bucket_add_one(Datum timeval, int64 bucket_width, Oid timetype)
```

Comment:
I had to read this a few times to realize it didn't add one to the time bucket but added one bucket (width) to the time.
```suggestion
ts_time_add_one_bucket(Datum timeval, int64 bucket_width, Oid timetype)
```

Reply:
@mkindahl Your suggestion violates the convention, where extern functions are prefixed with the module name, which is `time_bucket` as in the file name. So it can be:
```suggestion
ts_time_bucket_add_one_bucket(Datum timeval, int64 bucket_width, Oid timetype)
```